<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="griffon.exceptions.InstanceMethodInvocationExceptionSpec" tests="3" skipped="0" failures="0" errors="0" timestamp="2016-11-17T21:56:07" hostname="testing-gce-a500a3b7-3e8b-4697-b127-d17b45a2fbc5" time="0.014">
  <properties/>
  <testcase name="Exception should have proper formatted message (1)[0]" classname="griffon.exceptions.InstanceMethodInvocationExceptionSpec" time="0.006"/>
  <testcase name="Exception should have proper formatted message (1)[1]" classname="griffon.exceptions.InstanceMethodInvocationExceptionSpec" time="0.001"/>
  <testcase name="Exception should have proper formatted message (2)[0]" classname="griffon.exceptions.InstanceMethodInvocationExceptionSpec" time="0.007"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
